test/CoreTest.cpp

```diff
@@ -841,3 +841,53 @@ TEST_CASE( "no_file_ending", "[core]" )
     REQUIRE_THROWS_WITH(Series("./new_openpmd_output_%05T", Access::CREATE),
                         Catch::Equals("Unknown file format! Did you specify a file ending?"));
 }
+
+TEST_CASE( "custom_geometries", "[core]" )
+{
+    std::vector< int > sampleData( 10, 0 );
+    {
+        Series write( "../samples/custom_geometry.json", Access::CREATE );
+        auto E = write.iterations[ 0 ].meshes[ "E" ];
+        E.setAttribute( "geometry", "customGeometry" );
```

Comment:
```suggestion
        E.setAttribute( "geometry", "other:customGeometry" );
```

Reply:
I've left the test this way on purpose to demonstrate that `setAttribute` performs no automatic coercion and `setGeometry` should be used for that purpose.